docs/integrations/azure/locations.md

```diff
@@ -52,13 +76,19 @@ integrations:
 
 The configuration is a structure with these elements:
 
-- `host`: The DevOps host; only `dev.azure.com` is supported.
-- `token` (optional): A personal access token as expected by Azure DevOps.
-- `credential`: (optional): A service principal or managed identity
+- `credentials`: (optional): A service principal or managed identity
+
+The `credentials` element is a structure with these elements:
+
+- `organizations`: (optional): A list of organizations for which this credential should be used. Only supported for Azure DevOps (cloud). If not specified the credential will be used for all organizations for which no other credential is configured.
+- `clientId`: The client ID of the service principal or managed identity (required for service principal and managed identities)
+- `clientSecret`: The client secret of the service principal (required for service principal)
+- `tenantId`: The tenant ID of the service principal (required for service principal)
+- `token`: The personal access token (required for personal access token)
 
 > Note:
 >
-> - `token` and `credential` are mutually exclusive.
 > - You cannot use a service principal or managed identity for Azure DevOps Server (on-premises) organizations
 > - You can only use a service principal or managed identity for Azure AD backed Azure DevOps organizations
+> - You cannot use more than one organization with a personal access token
```

Comment:
Any reason for this?  When you create a PAT, you can generate a PAT for more than one organisation
![image](https://github.com/backstage/backstage/assets/289860/a7288681-ad49-42a8-8aad-963b5bb29960)

Reply:
I don't remember exactly why I added this restriction, but you are right it seems one or all organizations are allowed when creating a PAT, so I removed the restriction from the documentation and code.